packages/categorize/configure/src/design/index.jsx

```diff
@@ -136,6 +136,17 @@ export class Design extends React.Component {
     return countInAnswer(choice.id, model.correctResponse);
   };
 
+  checkAllowMultiplePlacements = (allowMultiplePlacements, c) => {
+    if (allowMultiplePlacements === 'Yes') {
+      return 0;
+    }
+    if (allowMultiplePlacements === 'No') {
+      return 1;
+    }
+    return c.categoryCount || 0;
+
+  };
```

Comment:
This can be simplified by checking the 'Set per choice' value first. Something like this: 
`if (allowMultiplePlacements === 'Set Per Choice') return c.categoryCount || 0; 
return allowMultiplePlacements === 'No'`